scripts/sync-to-aws-eks-charts.sh

```diff
@@ -153,13 +153,16 @@ cp -R "${helm_chart_path}/" "${eks_charts_nth_path}/"
 
 echo -e "🥑 Commit updates"
 
+git config --local user.name "ec2-bot 🤖"
+git config --local user.email "ec2-bot@users.noreply.github.com"
```

Comment:
This is to cause git to set "committer" and not just "author" on the commit, yes?

Reply:
The `git commit` command errored because the identity was not set.  Now that it *is* set, there is no need to explicitly specify the author of the commit.